Task4/db.go

```
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/go-redis/redis"
+)
+
+var rdb *redis.Client
+
+// init redis config
+func initClient() (err error) {
+	config, err := getRedisConfig()
+	if err != nil {
+		log.Printf("Get config file error: %v", err)
```

should raise the `err` out instead of logging only